build/build.js

```diff
@@ -35,6 +35,11 @@ exec('rollup -c -f umd -o umd/reactions-component.min.js', {
   NODE_ENV: 'production',
 });
 
+console.log('\nPlacing TypeScript definitions ...');
+
+exec('cp src/index.d.ts .')
```

Comment:
why bother with copying. You can place all definitions in typings directory directly. I do not see any win in colocating file and signature